bin/filter_gtf.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
```

Comment:
License/author is missing

Reply:
I believe this has been raised by you before (there are no new scripts, I just fixed up and renamed a couple). @drpatelh has this on the next milestone to fix https://github.com/nf-core/rnaseq/issues/976

Reply:
Added in https://github.com/nf-core/rnaseq/pull/1114- will merge shortly

Reply:
😆 I knew it felt familiar